impls/node/client.ts

```diff
@@ -31,7 +31,21 @@ bindLogger(
 );
 
 const clientTransport = new WebSocketClientTransport(
-  () => Promise.resolve(new WebSocket(`ws://${RIVER_SERVER}:${PORT}`)),
+  () =>
```

Comment:
you shouldn't need to do this, river already retries this ws getter internally

Reply:
i have some news for you, ha. we _do_ need to do this, because it is an unhandled exception.

lemme add a comment to avoid future surprises.

Reply:
oh wait, you meant the retry loop. that we can probably get rid of, yes.

Reply:
we attach an error listener on the ws already so errors shouldn't error out unless it fails at construction time (which should only happen on malformed urls?)

Reply:
before i added `/healthz` it was very common while the server was warming up but hadn't started listening on the port itself.

even after `/healthz`, it still happens regularly after a reconnect.

Reply:
oh wait, i see why: we _do not_ attach an error listener, ha https://github.com/replit/river/blob/main/transport/impls/ws/server.ts

this is why we need to run this as an UDS server. we don't use this configuration in production :upside_down_face: